.github/workflows/ci.yml

```diff
@@ -168,6 +168,14 @@ jobs:
         run: just init-db
       - name: Rebuild All
         run: just build-all
+      - name: Check No SCM Changes
+        run: |
+          if [[ -n $(git status -s) ]]
+          then
+            echo "Running the build resulted in changes to git controlled files:"
+            git --no-pager diff
+            exit 1
+          fi
```

Comment:
Nice! This is great. I also notice that `lsp/hoveritems.go` changes pretty often when I `just clean` and `just build-all` locally, so hopefully this will catch that case as well 👍